.github/workflows/ci.yml

```diff
@@ -15,4 +15,26 @@ jobs:
       - name: Install
         run: npm ci
       - name: Lint
-        run: npm run lint
\ No newline at end of file
+        run: npm run lint
+
+  postman:
+    runs-on: ubuntu-latest
+    # With the exception of GITHUB_TOKEN, secrets are not passed to the runner
+    # when a workflow is triggered from a forked repository.
+    if: github.repository == github.event.repository.full_name
+    steps:
+      - name: Check out git repository
+        uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - name: Install dependencies
+        run: npm ci
+        working-directory: packages/tests-postman
+      - name: Run Postman tests
+        run: node .
+        working-directory: packages/tests-postman
+        env:
+          CLIENT_SECRET_VC_MESUR_IO: ${{ secrets.CLIENT_SECRET_VC_MESUR_IO }}
+          CLIENT_SECRET_b0643936e30843b622c78ad353da19123e4583817491fe05e2e9d39824983bee: ${{ secrets.CLIENT_SECRET_b0643936e30843b622c78ad353da19123e4583817491fe05e2e9d39824983bee }}
```

Comment:
similar to my comment on the other PR, would much prefer this to be `CLIENT_SECRET_TRANSMUTE`